generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
+
+	"github.com/shurcooL/go/vfs/httpfs/filter"
+	"github.com/shurcooL/vfsgen"
+)
+
+func main() {
+	ignore := func(fi os.FileInfo, path string) bool {
+		switch {
+		case len(fi.Name()) >= 2 && fi.Name()[0] == '.': // Skip .files/.folders (but not ".").
+			return true
+		case pathpkg.Ext(fi.Name()) == ".go":
+			return true
+		case path == "/README.md":
+			return true
+		default:
+			return false
+		}
+	}
+	var data http.FileSystem = filter.NewIgnore(http.Dir("."), ignore)
+
+	err := vfsgen.Generate(data, vfsgen.Options{
+		Filename:     "data.go",
+		PackageName:  "data",
+		VariableName: "Assets",
+	})
+	if err != nil {
+		log.Fatalln(err)
+	}
+}
```

Comment:
> but VariableName needs to stay because the default is assets (since there's no reason to have an exported identifier inside non-library Go packages), but it needs to be exported in appdash-data. In fact, I've renamed it to Data.

I would argue there is _no reason to not have it exported by default_, given that nobody will import `main` packages anyway so that only leaves library packages. But if you disagree then it's A-OK to keep as-is.

> perhaps I should change the default to be {{toLower .VariableName}}_vfsdata.go. Do you think that's a good idea?

I think so, yes. Having capital letters in filenames is weird.

> Sorry, which naming are you unsure about? I've proposed both assets and data, but I think data looks nicer since it's a better fit with the repo name. What would you like the naming to be?

Data works best, yeah. I was confused, please ignore that paragraph :).